meson.build

```diff
@@ -328,7 +328,7 @@ if get_option('gcc_vector')
 #   error "GCC vector intrinsics are disabled on GCC prior to 4.9"
 # elif defined(__arm__)
 #   error "GCC vector intrinsics are disabled on ARM"
-# elif !defined(__x86_64__)
+# elif (__SIZEOF_POINTER__ < 8)
```

Comment:
This might catch the x32 ABI. Not that many people actually use it…

Is there a particular architecture symbol for ppc64le that we can use, instead?

Reply:
we really should use vector support on every architecture that can support it, so changing things like that is a non-fix... if gcc vectors can work on 32-bit targets i'd just drop the check entirely, otherwise we could extend this check to like: `#elif (__SIZEOF_POINTER__ < 8 && !defined(__x86_64__))`